### PR TITLE
Make config synchronizer part of pansophy

### PR DESCRIPTION
**Pull request by mnoack**
- Saves copying it to all projects
- Just specify ENV['VERSION'](defaults to 0)
- Including app just says in Rakefile: require 'pansophy/tasks'

### CODE REVIEW

**Comment by msmirnoff** [comment]
:+1:

File: lib/pansophy/config_synchronizer.rb [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+  def pull
+    verify_app_name!
+    Pansophy.pull(app_name, remote_path, local_path, overwrite: true)
+  end
+
+  private
+
+  def app_name
+    ENV['AUDITOR_APP_NAME']
+  end
```
Comment (adz) [inline lib/pansophy/config_synchronizer.rb]: Funny env var?  Did you mean app_name

File: lib/pansophy/config_synchronizer.rb [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+  memoize :app_name
+
+  def remote_path
+    Pathname.new('config').join(version.to_s)
+  end
+
+  def version
+    ENV['CONFIG_VERSION'] || '1.0'
+  end
```
Comment (adz) [inline lib/pansophy/config_synchronizer.rb]: This is present in the including project,not env

File: lib/pansophy/config_synchronizer.rb (hunk 1 above)
Comment (mnoack) [inline lib/pansophy/config_synchronizer.rb]: Whoops. Fixed and force pushed.

File: lib/pansophy/config_synchronizer.rb (hunk 2 above)
Comment (mnoack) [inline lib/pansophy/config_synchronizer.rb]: Yes, I was planning on having this in Elastic beanstalk's env, so if we do a new config we can change it there, e.g. say we want to switch mail servers without deploying.

Currently we have to edit the sync task on the project, change config then deploy,

**Comment by msmirnoff** [comment]
:+1:

**Comment by mnoack** [comment]
@AlessandroBerardi Addressed all comments

**Comment by msmirnoff** [comment]
:+1:

**Comment by alxberardi** [comment]
Waiting for Coveralls to complete @mnoack , then :+1:

**Comment by msmirnoff** [comment]
:+1:

**Pull request merged**